hooks/before_vm_start/50_hana

```diff
@@ -42,6 +42,18 @@ if 'hana' in os.environ:
             cpu.appendChild(feature_rdt)
             cpu.appendChild(feature_lvl3)
 
+        if len(domain.getElementsByTagName('clock')):
+            clock = domain.getElementsByTagName('clock')[0]
+            tscClock = domxml.createElement('clock')
+            tscClock.setAttribute('offset', 'utc')
+            timer = domxml.createElement('timer')
+            timer.setAttribute('name','tsc')
+            # Adjust frequency to match the lowest value in your cluster
+            timer.setAttribute('frequency','2494140000')
```

Comment:
Probably deriving the value from the host itself makes most sense instead of hardcoding it.
This would mean we have the same frequency throughout the cluster (which could be a prerequisite).
The ideal solution would be getting the frequency from vdsm stats.